serverlessworkflow/modules/ROOT/pages/cloud/operator/build-and-deploy-workflows.adoc

```diff
@@ -131,6 +135,138 @@ spec:
 
 This parameters will only apply to new build instances.
 
+=== Passing arguments to the internal builder
+
+You can pass build arguments (see link:{docker_doc_arg_url}[Dockerfile ARG]) to the `SonataFlowBuild` instance.
+
+.Procedure
+
+1. Create or edit an existing `SonataFlowBuild` instance. It has the same name of the `SonataFlow` you're trying to build.
++
+.Checking if the SonataFlowBuild instance already exists
+[source,bash,subs="attributes+"]
+----
+# Check if the build exists
+kubectl get sfbuild/<name> -n <namespace>
+----
++
+2. Add to the `.spec.buildArgs` the desired arguments.
++
+.Adding buildArgs to the SonataFlowBuild instance
+[source,yaml,subs="attributes+"]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowBuild
+metadata:
+  name: <name>
+spec:
+  buildArgs:
+    - name: ARG1
+      value: value1
+    - name: ARG2
+      value: value2
+----
++
+3. Save the file and apply it to the cluster. A new build should start soon with the new configuration.
+
+Alternatively, you can set this information in the `SonataFlowPlatform`, so that every new build instances will use it as a template. For example:
```

Comment:
```suggestion
Alternatively, you can set this information in the `SonataFlowPlatform`, so that every new build instance will use it as a template. For example:
```